onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -122,6 +122,10 @@ void BasicBackend::PopulateConfigValue(ov::AnyMap& device_config) {
     device_config.emplace(ov::enable_profiling(true));
   }
 #endif
+
+  // Set a priority level for the current workload for preemption;  default priority is "DEFAULT"
+  device_config.emplace(ov::hint::model_priority(global_context_.model_priority));
```

Comment:
Does it make sense to expose the entire `config` to be specified by user, instead of exposing config options separately?

Reply:
We are exploring the means to pass a config file that can set / override defaults of such OpenVINO properties and hints but there is also value in exposing them individually like this as it lets someone quickly change some important and frequently used settings through OVEP provider options without the need for a config file.

Reply:
In fact, by `config` I meant the dictionary that could be provided into `Core::compile_model`
(See: https://docs.openvino.ai/2022.3/classov_1_1Core.html) - this is the way to specify plugin specific options in OpenVINO.

For example:
```
Core.compile_model(model, device, { "PERFORMANCE_HINT": "THROUGHPUT" }); // specify the performance hint
Core.compile_model(model, device, { "MODEL_PRIORITY": "HIGH" }); // set the priority 
Core.compile_model(model, device, { "LOG_LEVEL": "DEBUG" }); // any other options supported by particular plugin
```

For configuring `OVEP`, user can simply provide the entire config instead of setting options separately:
```
OrtOpenVINOProviderOptions options{};
options.config = { {"MODEL_PRIORITY": "HIGH"}, {"PERFORMANCE_HINT": "THROUGHPUT"}, ... };
session_options->AppendExecutionProvider_OpenVINO(options)
```

The benefits of this approach is that there is no need to change the existing code in both `OVEP` and `User` sides if there is a need to support new options.

Reply:
That makes sense, and we'll implement this in a later PR and only expose model priority in the scope of this one. Model priority still needs to be a provider option in OVEP.